### ISSUE TITLE
Can't login to ChatGPT.com with this extension active using Brave web browser

### ISSUE THREAD

**Issue by umasse** [josStorer/chatGPTBox#794]
**Describe the bug**
**问题描述**
When I am logged out of ChatGPT, I need to disable this extension in order to log in.

**To Reproduce**
**如何复现**
1. Go to ChatGPT.com. If you are logged in, log out first.
2. Try to login.
3. See error:
```
OpenAI
Oops!, something went wrong
There could be a misconfiguration in the system or a service outage. We track these errors automatically, but if the problem persists feel free to contact us.
Please try again.
TECHNICAL DETAILS
invalid_request: You may have pressed the back button, refreshed during login, opened too many login dialogs, or there is some issue with cookies, since we couldn't find your session. Try logging in again from the application and if the problem persists please contact the administrator.
```

**Expected behavior**
**期望行为**
Simply being able to login.

**Screenshots**
**截图说明**
![image](https://github.com/user-attachments/assets/14ca42d2-3d1e-4dea-9969-b6c342aa76f3)

**Please complete the following information):**
**请补全以下内容**
 - OS: Ubuntu 24.04
 - Browser: Brave browser Version 1.70.119 Chromium: 129.0.6668.70 (Official Build) (64-bit)
 - Extension Version: 2.5.8

**Comment by josStorer** [comment]
Not sure whether this error is directly related to this project. I've noticed some discussions about it on Reddit as well: https://www.reddit.com/r/OpenAI/comments/1fqtbp9/cant_log_in_to_chatgpt/

Has anyone else encountered this issue? If so, please comment below this issue to let me know. Also welcome any potential solutions you might have.

**Comment by umasse** [comment]
It could be that other extensions are also triggering the problem, but I can reproduce it right now, 100% consistently:
- No extensions activated: I can login
- Only ChatGPTBox activated: Error

It's not a big problem. Once I log in, I turn the extension back on, and it stays working fine (until the OpenAI cookie expires, I assume).

**Comment by josStorer** [comment]
I'm not yet sure of the exact reason for this issue. The restriction policy might vary by location. In my case, I haven't encountered any login restrictions while using ChatGPTBox

**Comment by zzkcode** [comment]
Same issue here, login in always encounter 'something went wrong' with ChatGPTBox extension enabled.

**Comment by umasse** [comment]
Yes, it's still going on. I just experienced it today. I added the "Extensity" extension, which allows you to quickly turn on/off other extensions, so I can easily work around it.

Does the extension modify anything in the Login page? Any tips on where to look to try to get more info for troubleshooting?

**Comment by Mohamed3nan** [comment]
Yeah, same. To resolve the issue, disable the extension, log in, and then re-enable it..
